configlib/config.go

```diff
@@ -1,41 +1,42 @@
 package configlib
 
 import (
-	log "github.com/sirupsen/logrus"
-	"github.com/spf13/afero"
-	"github.com/spf13/viper"
-	"gopkg.in/yaml.v2"
 	"os"
 	"path"
 	"path/filepath"
 	"runtime"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
 )
 
 //Whenever
 var AvailableConfiguration Config
 var ConfigurationLoaded bool = false
 
 type Config struct {
-	NMVersion       string                  `yaml:"nmVersion" json:"nm_version,omitempty"`
+	NMVersion       string                  `mapstructure:"nm_version" yaml:"nmVersion" json:"nm_version,omitempty"`
```

Comment:
We'll want to make sure the yaml keys stay the same for the sake of cleanliness.